cmd/dep/testdata/gb/golden.txt

```diff
@@ -0,0 +1,6 @@
+Detected gb manifest file...
+Converting from gb manifest...
+  Using ^2.0.0 as initial constraint for imported dep github.com/sdboyer/deptestdos
```

Comment:
Once the constraint logic is fixed, this will change to `Using master as initial constraint`.